index.css

```diff
@@ -414,3 +414,155 @@ hr {
   margin-bottom: 20px;
   padding: 0;
 }
+
+.nav-bar-desk {
+  display: none;
+}
+
+/* Media queries for desktop view */
+
+@media screen and (min-width: 768px) {
+  .nav-bar-desk {
+    display: flex;
+  }
+
+  .main-section {
+    display: flex;
+    flex-direction: column;
+    margin: 5% 260px 325px;
+  }
+
+  .three-bar-menu {
+    display: none !important;
+  }
+
+  body {
+    background-size: 95%;
+    background-position-y: 60px;
+    background-position-x: 34.6px;
+  }
+
+  .header-bar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .header-bar header {
+    padding-left: 100px;
+  }
+
+  .nav-bar-list {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .ul-desk {
+    display: flex;
+  }
+
+  li {
+    padding: 0 20px;
+  }
+
+  li a {
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+  }
+
+  li:last-child {
+    padding-right: 12px;
+  }
+
+  .cards {
+    flex-direction: row;
+    height: 452px;
+    margin: 114px 8% 15%;
+  }
+
+  .cards .second-container {
+    margin-left: 0;
+    margin-right: 0;
+  }
+
+  .cards .second-container p {
+    max-width: 100%;
+  }
+
+  .main-img img {
+    width: 465px;
+    height: 400px;
+    margin: 0 14px 0;
+  }
+
+  .cards article .horizontal-section {
+    justify-content: flex-start;
+  }
+
+  .horizontal-section > * {
+    padding-right: 10px;
+    margin-left: 0;
+  }
+
+  .card-2 .main-img,
+  .card-4 .main-img {
+    order: 1;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: row;
+    margin-right: 0;
+  }
+
+  .about-section-1 {
+    flex: 1;
+  }
+
+  .about-section-2 {
+    padding-top: 80px;
+    flex: 1.5;
+  }
+
+  .skills-images {
+    display: flex;
+    flex-direction: row;
+    padding: 0;
+  }
+
+  .skills-boxed {
+    display: flex;
+    flex-direction: column;
+    width: 100px;
+    justify-content: center;
+    height: 100px;
+    font-size: 13px;
+  }
+
+  .see-resume {
+    padding-left: 2px;
+    padding-right: 2px;
+  }
+
+  .form-container-1,
+  .form-container-2 {
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    width: 59%;
+    height: 50%;
```

Comment:
- [ ] The background image in your `form section` appears to be colliding with the input elements. Kindly adjust its position so that the input elements are clearly visible.
![image](https://user-images.githubusercontent.com/98453979/220968917-d6da2470-dd7e-44f5-9970-56fc6441a339.png)